.gitignore

```diff
@@ -15,4 +15,5 @@ erizo_controller/erizoAgent/erizo-*.log
 .project
 .cproject
 spine/erizofc.js
-extras/basic_example/public/assets/
+spine/spineClientsConfig.json
```

Comment:
This will cause problems, as it is the default file used by `runSpineClients`, if you want to make changes for your tests you should create a new file and use `runSpineClients -s config_file.js`